style.css

```diff
@@ -18,4 +18,9 @@ Text Domain: planet4-child-theme-greece
     font-size: calc(100% + 2vw);
     width: 90%;
   }
+
+  .happy-point-block-wrap {
+    iframe {
```

Comment:
I think nesting won't work here. Remember, we don't have Sass support on child themes. So this should probably be `.happy-point-block-wrap iframe`.